attributes/default.rb

```diff
@@ -18,18 +18,19 @@
 # limitations under the License.
 #
 
-default['push_jobs']['package_url']      = nil
-default['push_jobs']['package_checksum'] = ''
-default['push_jobs']['gem_url']          = nil
-default['push_jobs']['gem_checksum']     = ''
-
-default['push_jobs']['whitelist']   = { 'chef-client' => 'chef-client' }
+default['push_jobs']['package_url']           = nil
+default['push_jobs']['package_checksum']      = ''
+default['push_jobs']['gem_url']               = nil
+default['push_jobs']['gem_checksum']          = ''
+default['push_jobs']['whitelist']             = { 'chef-client' => 'chef-client' }
+default['push_jobs']['cookbook']              = 'push-jobs'
+default['push_jobs']['environment_variables'] = {}
```

Comment:
Made changed in https://github.com/runwaldarshu/push-jobs/commit/cdde557030436fe7018568f5ff5135edf4fc9642